lib/octicons_node/package.json

```diff
@@ -17,9 +17,8 @@
     "url": "https://github.com/primer/octicons/issues"
   },
   "scripts": {
-    "publish": "../../script/notify success",
```

Comment:
Should we keep these in here until the publish stuff is merged in?

Reply:
Actions notifies for us now. 🥇